lib/tasks/verify_historic_duplicates.rake

```diff
@@ -0,0 +1,12 @@
+task verify_historic_duplicates: :environment do
+  Statement.original.each do |statement|
+    next unless statement.duplicate_statements.any?
```

Comment:
This check isn't needed as we're looping through the `duplicate_statements` so we won't be doing anything if empty already.